test/functional/services/dashboard/panel_actions.ts

```diff
@@ -253,48 +263,28 @@ export class DashboardPanelActionsService extends FtrService {
   }
 
   async openInspector(parent?: WebElementWrapper) {
-    await this.openContextMenu(parent);
-    const exists = await this.testSubjects.exists(OPEN_INSPECTOR_TEST_SUBJ);
-    if (!exists) {
-      await this.clickContextMenuMoreItem();
-    }
-    await this.testSubjects.click(OPEN_INSPECTOR_TEST_SUBJ);
+    await this.clickContextMenuItem(OPEN_INSPECTOR_TEST_SUBJ, parent);
   }
 
-  async legacyUnlinkFromLibary(parent?: WebElementWrapper) {
+  async legacyUnlinkFromLibrary(parent?: WebElementWrapper) {
     this.log.debug('legacyUnlinkFromLibrary');
-    await this.openContextMenu(parent);
-    const exists = await this.testSubjects.exists(LEGACY_UNLINK_FROM_LIBRARY_TEST_SUBJ);
-    if (!exists) {
-      await this.clickContextMenuMoreItem();
-    }
-    await this.testSubjects.click(LEGACY_UNLINK_FROM_LIBRARY_TEST_SUBJ);
+    await this.clickContextMenuItem(LEGACY_UNLINK_FROM_LIBRARY_TEST_SUBJ, parent);
     await this.testSubjects.waitForDeleted(
       'embeddablePanelNotification-ACTION_LIBRARY_NOTIFICATION'
     );
   }
 
   async unlinkFromLibary(parent?: WebElementWrapper) {
```

Comment:
since you already fixed this type elsewhere, you can probably fix also this typo `Libary` :D

Reply:
Done 89cb0b6